_installation/interoperability.md

```diff
@@ -0,0 +1,112 @@
+**Default collations**
+
+Default collations are different in the T-SQL and PL/pgSQL, and this may cause some semantic differences. For example, `'a' = 'A'` is true by default in Babelfish T-SQL as the default collation is case-insensitive; but this will be false by default in PL/pgSQL where the collation is case-sensitive. 
+
+**Future Babelfish upgrades**
+
+The most likely reason you might want to implement part of a migrated T-SQL application in PostgreSQL, is to work around a particular piece of T-SQL functionality that is not currently supported by Babelfish. There is a risk that, if Babelfish starts to support those features in a future release, the application would not be compatible with the future release. This could mean that the upgraded future instance may not work correctly, or will cause the upgrade to fail.
+
+
+### Interoperability scenarios
+
+The simplest interoperability scenario is to vertically split an application into a T-SQL part and a PostgreSQL part, each with their own users and schemas and no interaction between them. The application opens both a TDS connection to the Babelfish database, and a PostgreSQL connection to a separate PostgreSQL database. In this case, there is in fact no interoperability issue on a database level, since the PostgreSQL access and T-SQL access are fully separate and do not interfere. Note that there may be a dependency if both connections use the same username to connect.
+
+A more relevant variation is where the PostgreSQL connection accesses the Babelfish database, but accesses only PostgreSQL-created objects, while T-SQL is used to access only Babelfish-created objects (such as tables, views, and procedures). In this case there is no interference, but in practice there may be a requirement to exchange information between the T-SQL context and the PostgreSQL context, for example, through a commonly accessed table. Such a table should be created in T-SQL rather than in PostgreSQL, but otherwise this scenario should not be problematic. In this case, you may encounter interoperability aspects in schema names, identifiers, and permissions.
+
+A case with few expected complications is using the PostgreSQL connection to read from Babelfish-created tables. This scenario can potentially run into problems with schema names, identifiers, and permissions. When reading from views, you may also run into additional considerations related to schema names and object identifiers referenced by views.
+
+Note that when the PostgreSQL connection modifies Babelfish-created tables or performs DDL, you may run into problems with transactional aspects of the semantics in use.
+
+A more complex scenario is when a PostgreSQL connection executes a T-SQL procedure (either directly or via a PostgreSQL procedure), or modifies a Babelfish-created table which has a Babelfish-created trigger. You should carefully evaluate both the semantics in use and the result set to ensure that the behavior is what you expect. 
+
```

Comment:
What happened to the bullet about DMS?  Did Bill Ramose recommend to remove it? If not, we need to keep it in, since this point has come up frequently